jdbc/src/test/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaDataTest.java

```diff
@@ -255,6 +308,10 @@ private void initializeWithResult() throws SQLException {
     Mockito.when(dbResultSet.next()).thenReturn(true).thenReturn(false);
     Mockito.when(dbResultSet.getString(1)).thenReturn("testDB");
     Mockito.when(mockStatement.executeQuery("SHOW DATABASES")).thenReturn(dbResultSet);
+    Mockito.when(mockStatement.executeQuery("SHOW DATABASES LIKE '%'")).thenReturn(dbResultSet);
+    Mockito.when(mockStatement.executeQuery("SHOW DATABASES LIKE '%test%'")).thenReturn(dbResultSet);
+    Mockito.when(mockStatement.executeQuery("SHOW DATABASES LIKE 'testDB'")).thenReturn(dbResultSet);
+    Mockito.when(mockStatement.executeQuery("SHOW DATABASES LIKE '%testDB%'")).thenReturn(dbResultSet);
```

Comment:
1/ Add a test case that returns empty Resultset
2/ Add a test case that throw could exception "Access denied" to simulate user who does not have  Timestream Read permissions
3/ Add a test case that could return multiple rows and may require pagination

Reply:
Test 2 for "Access denied" is added.